packages/nimble-components/src/table/models/table-validator.ts

```diff
@@ -1,10 +1,21 @@
-import { TableRecord, TableRowSelectionMode, TableValidity } from '../types';
+import {
+    TableRecord,
+    TableRowSelectionMode,
+    TableValidity,
+    Validatable,
+    ValidityObject
+} from '../types';
+
+export interface Validator {
```

Comment:
Do we need both a `Validator` and a `Validatable` interface? They look very similar, so it isn't clear to me when a class should implement one vs the other.

Reply:
We actually don't need either of them anymore. Removed both.